File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        int temp=x;
        long long num=0;
        while(temp>0){
            num=num*10+(temp%10);
            temp/=10;

        }if(num==x){
            return true;
        }else{
            return false;
        }
        
    }
};

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        new=0
        temp=x
        while x>0:
            num=x%10
            new=new*10+num
            x//=10
        if new==temp:
            return True
        else:
            return False

        